pkgs/tools/virtualization/ovftool/default.nix

```diff
@@ -119,14 +158,16 @@ stdenv.mkDerivation rec {
     # binary wrapper; note that LC_CTYPE is defaulted to en_US.UTF-8 by
     # VMWare's wrapper script. We use C.UTF-8 instead.
     install -m 755 -d "$out/bin"
-    makeWrapper "$out/libexec/${pname}/ovftool.bin" "$out/bin/ovftool" \
+    makeWrapper "$out/libexec/${pname}/$ovftool_bin" "$out/bin/ovftool" \
       --set-default LC_CTYPE C.UTF-8 \
-      --prefix LD_LIBRARY_PATH : "$out/lib"
+      --prefix ${if stdenv.isDarwin then "DYLD_LIBRARY_PATH" else "LD_LIBRARY_PATH"} : ${if stdenv.isDarwin then "$out/lib/ovftool" else "$out/lib"}
```

Comment:
I have no idea how to handle the dylib version of autoPatchelf on OS X. This builds and iI get a bunch of this:

/nix/store/7i52bc3x91b5pnrjpnwf80b1px41dlbb-ovftool-4.5.0-20459872/lib/ovftool/libssl.1.0.2.dylib

But I get this:

```
dyld: Library not loaded: @loader_path/lib/libcares.2.dylib
  Referenced from: /nix/store/7i52bc3x91b5pnrjpnwf80b1px41dlbb-ovftool-4.5.0-20459872/libexec/ovftool/ovftool
  Reason: image not found
```